app/controllers/concerns/spina/current_spina_account.rb

```diff
@@ -0,0 +1,28 @@
+module Spina
+  module CurrentSpinaAccount
+    extend ActiveSupport::Concern
+    
+    included do
+      before_action :current_spina_account
+      helper_method :current_spina_account
+      helper_method :current_account
```

Comment:
Any way we can add the same conditional here?

Reply:
I tried that originally, putting an unless at the end of `helper_method :current_account` but it seemingly wasn't working that way. I agree just wrapping the whole method in an if statement is ugly, but it was the only way I could see to disable it. Any suggestions on this would be appreciated :)